### DISCUSSION TITLE
realtime subscription api

### DISCUSSION THREAD

**Discussion by atiframlee** [supabase/supabase#3449] [Questions] [answered]
previously i have already applied the subscribe to realtime api.
i placed it inside my useEffect()

 useEffect(() => {
       const mySubscription = supabase
    .from('*')
    .on('*', payload => {
      console.log('Change received!', payload)
    })
    .subscribe()

    }, [])

and it works just fine, now with the similar syntax it seems no longer working, whenever i update my table, i do not receive any console.log message/ payload.i already try toggling the replication on the setting section on and off and still no affect.

**Comment by w3b6x9** [comment, marked as answer]
@atiframlee then this might be specific to your Supabase project. Can you please email support@supabase.io with a link to this discussion and your project ref please? We can investigate!
